File: Utt2Database/Player.vb
Imports System
Imports System.Collections.Generic
Imports Microsoft.VisualBasic

Namespace Utt2Database
    Partial Public Class Player
        Public Property Id As Long

        Public Property Slug As String

        Public Property Name As String

        Public Property SkinData As String

        Public Property Country As String

        Public Overridable Property PlayerLogs As ICollection(Of PlayerLog) = New List(Of PlayerLog)()

        Public Overridable Property PlayerStats As ICollection(Of PlayerStat) = New List(Of PlayerStat)()
    End Class
End Namespace

File: src/IndexerSharp/Utt2Database/Player.cs
namespace Naomai.UTT.Indexer.Utt2Database;
public partial class Player
{
    public long? Id { get; set; }
    public string Slug { get; set; } = null!;
    public string Name { get; set; } = null!;
    public string SkinData { get; set; } = null!;
    public string? Country { get; set; } = null!;
    public virtual ICollection<PlayerLog> PlayerLogs { get; set; } = new List<PlayerLog>();
    public virtual ICollection<PlayerStat> PlayerStats { get; set; } = new List<PlayerStat>();

    public override string ToString()
    {
        return "P" + Id + "#" + Slug + "[" + Name + "]";
    }
}